db/seeds.rb

```diff
@@ -49,14 +49,24 @@
 
   # Could have done in the above loop, but that was getting a bit messy
   puts 'Creating Items in the API, by Item Type'
+  items = YAML::load_file(File.join(Rails.root, 'db/db_yml', 'items.yml'))
+  itemsint = 0
   ItemType.all.each do |item_type|
     if Inventory.items_by_type(item_type.uuid).empty?
       (1..10).each do |i|
-        Inventory.create_item(item_type.uuid, "#{item_type.name} Cart #{i}", true, {})
+        Inventory.create_item(item_type.uuid, items[itemsint]["name"], true, {})
+        itemsint += 1
       end
     end
   end
 
+  puts 'Pulling Items from Api and Storing Locally'
+  ItemType.all.each do |item_type|
+    items = Inventory.items_by_type(item_type.uuid)
+    items.each do |item|
```

Comment:
Do have enough char space (style wise) to put this on one line `Inventory.items_by_type(item_type.uuid).each do |item|`?